src/utils/getInsightImportanceColor.ts

```diff
@@ -0,0 +1,46 @@
+import { DefaultTheme } from "styled-components";
+
+export const getInsightImportanceColor = (
+  importance: number,
+  theme: DefaultTheme
+): string | undefined => {
+  if (importance === 0) {
+    return undefined;
+  }
+
+  if (importance < 3) {
+    switch (theme.mode) {
+      case "light":
+        return "#f93967";
+      case "dark":
+      case "dark-jetbrains":
+        return "#e00036";
+    }
+  }
+  if (importance < 5) {
+    switch (theme.mode) {
+      case "light":
+        return "#e06c00";
+      case "dark":
+      case "dark-jetbrains":
+        return "#ff810d";
+    }
+  }
```

Comment:
it seems like values of 5, 6, 7 are not taken care of and falling to default.
is that the requirement?

Reply:
Good catch!